crates/bevy_input/src/gamepad/event.rs

```diff
@@ -0,0 +1,126 @@
+use crate::gamepad::{Gamepad, GamepadEventType};
+
+/// An event of a [`Gamepad`].
+///
+/// This event is the translated version of the [`GamepadEventRaw`].
+/// It is available to the end user and can be used for game logic.
+///
+/// ## Access
+///
+/// To access or send gamepad events use one of the following:
+/// - To access gamepad events: `EventReader<GamepadEvent>`
+/// - To send gamepad events: `EventWriter<GamepadEvent>`
+///
+/// ## Differences
+///
+/// The difference between the [`GamepadEventRaw`] and the [`GamepadEvent`] is that the latter respects
```

Comment:
IMO this should be moved above, where you explain the relation between the two event types.